CMakeLists.txt

```diff
@@ -63,6 +63,15 @@ macro(build_yaml_cpp)
   )
 endmacro()
 
+find_package(yaml-cpp 0.6 QUIET)
+if(yaml-cpp_FOUND)
+  message(STATUS "Found yaml-cpp ${yaml-cpp_VERSION}")
+  ament_package(
+    CONFIG_EXTRAS "yaml_cpp_vendor-extras.cmake.in"
+  )
+  return()
```

Comment:
return isn't a good idea, as you still want to run lines 92-94.

Reply:
Thanks for the feedback. I reordered the logic and hope it looks more appropriate.